ThinBridge/ThinBridge
# Add a script to update version number

# Which issue(s) this PR fixes:

N/A.

# What this PR does / why we need it:

Currently we need to update version number of each modules respectively, it's too bother.
It's convenient to add a script to update all version numbers.

In addition, we should apply same version number to each modules in future releases to avoid confusion.

It's based on @yashirot's work in https://gitlab.com/clear-code/browserselector/-/tree/e64d416a8fff577068783d2a9782a9caebdb5c8f/script

# How to verify the fixed issue:

Run the script then check `git diff` to verify that the script expectedly replace version numbers.

## The steps to verify:

1. Open PowerShell window (or something similar) at the script folder
2. Run ` powershell.exe -ExecutionPolicy Bypass -file .\ThinBridgeVersionUp.ps1 4.2.1.0`
3. Check `git diff`
4. Run unit test: `powershell.exe -ExecutionPolicy Bypass Invoke-Pester`

## Expected result:

> 3. Check `git diff`

Version numbers are replaced with `4.2.1.0`

> 4. Run unit test: `powershell.exe -ExecutionPolicy Bypass Invoke-Pester`

Pass all tests
